akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -265,6 +267,15 @@ private[akka] class RemoteActorRefProvider(
       remoteSettings.configureDispatcher(Props[RemoteDeploymentWatcher]()),
       "remote-deployment-watcher")
 
+  /** Can be overridden when using RemoteActorRefProvider as a superclass rather than directly */
+  protected def showDirectUseWarningIfRequired() = {
+    if (remoteSettings.WarnAboutDirectUse) {
+      log.warning(
+        "Using the 'remote' ActorRefProvider directly, which is a low-level feature intended for power users. " +
```

Comment:
Can we be even stronger and just say you shouldn't use it? Some ppl like being power users :)

Reply:
+1 on that